docs/tutorial/extensions.md

```diff
@@ -513,3 +513,33 @@ assert circuit.encrypt_run_decrypt(0, 3, -5) == -5
 {% hint style="info" %}
 `fhe.if_then_else` is just an alias for [np.where](https://numpy.org/doc/stable/reference/generated/numpy.where.html).
 {% endhint %}
+
+## fhe.identity(value)
+
+Allows you to copy the value:
+
+```python
+import numpy as np
+from concrete import fhe
+
+@fhe.compiler({"x": "encrypted"})
+def f(x):
+    return fhe.identity(x)
+
+inputset = [np.random.randint(-10, 10) for _ in range(10)]
+circuit = f.compile(inputset)
+
+assert circuit.encrypt_run_decrypt(0) == 0
+assert circuit.encrypt_run_decrypt(1) == 1
+assert circuit.encrypt_run_decrypt(-1) == -1
+assert circuit.encrypt_run_decrypt(-3) == -3
+assert circuit.encrypt_run_decrypt(5) == 5
+```
+
+{% hint style="info" %}
+Identity extension can change the bit-width of the input.
+{% endhint %}
+
+{% hint style="warning" %}
+Identity extension only works in `Native` encoding, which is usually selected when all table lookups in the circuit are below or equal to 8 bits.
+{% endhint %}
```

Comment:
Maybe some explanation on why this is useful could improve that?

Reply:
Maybe a flag could disable that fhe.identity(x, slow_crt_compatibility=True).
So only TLU and no lsb are used in this case.

Reply:
I don't think cluttering the API with that is a good idea. They can do that with univariate if necessary. Writing this is even more confusing IMO, like what is CRT?

Reply:
What about someone that want to test the same code with the two implementation native / crt ?
Is the native / crt available in configuration ?

Reply:
That's a separate issue, they can't do that with or without `slow_crt_compatibility` or not. If you think about it `slow_crt_compatibility` is about compatibility, it doesn't guarantee CRT will be used. So it can't be used for `test the same code with the two implementation native / crt ?`.

It's not, there is an issue for it I think.